.github/workflows/bot-coderabbit-gate.yml

```
@@ -0,0 +1,108 @@
+name: Bot — CodeRabbit Gate
+
+# Updates the coderabbit-review commit status based on the PR's current
+# CodeRabbit label. Also handles resolving/unresolving review threads.
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, synchronize]
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)